pymc/tests/test_smc.py

```diff
@@ -80,8 +83,9 @@ def test_sample(self):
         initial_rng_state = np.random.get_state()
         with self.SMC_test:
             mtrace = pm.sample_smc(draws=self.samples, return_inferencedata=False)
-        assert_random_state_equal(initial_rng_state, np.random.get_state())
-
+        assert_random_state_equal(
+            initial_rng_state, np.random.get_state()
+        )  # TODO: why this? maybe to verify that nothing was sampled?
```

Comment:
The PR where this test was added provides some context: https://github.com/pymc-devs/pymc/pull/5131

Since it was not obvious why it was there, a comment before the assert explaining the purpose of the check could be helpful.